Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN apt-get update && apt-get -y install git \
            unzip
 
 RUN mkdir /JavaPrograms
+RUN mkdir /JavaPrograms/scripts
 RUN mkdir /code
```

Comment:
yeah dawg! Check this out when you have time: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/ it's really interesting what an "image" actually is, think "sandwich layers" vs. "entire sandwich" :)